Automattic/jetpack
# MU WPCOM: Port the starter-page-templates feature from ETK

Fixes https://github.com/Automattic/dotcom-forge/issues/8242

## Proposed changes:

* Published NPM package [@automattic/page-pattern-modal](https://www.npmjs.com/package/@automattic/page-pattern-modal)
  * See https://github.com/Automattic/wp-calypso/pull/92871
* Ported and refactored the [starter-page-templates](https://github.com/Automattic/wp-calypso/tree/trunk/apps/editing-toolkit/editing-toolkit-plugin/starter-page-templates) feature from ETK
  * Use namespace `Automattic\Jetpack\Jetpack_Mu_Wpcom` to avoid collisions



### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:





* Follow the instructions below to use this branch in simple or atomic. I have pending to test on atomic
  * If that doesn't work on simple, make sure to run `bin/jetpack-downloader reset jetpack-mu-wpcom-plugin` first. 
  * Make sure the files are uploaded to the right folder (it can be `moon`, or `production`, or `sun`). 
  * Alternatively, you can use the rsync approach from local. Read more PCYsg-Osp-p2
* Go to ` { SITE }/wp-admin/post-new.php?post_type=page`
* Verify the assets are served from `/vendor/automattic/jetpack-mu-wpcom/src/build/`
* Verify the feature works as before

<img width="1728" alt="Screenshot 2567-07-25 at 18 58 16" src="https://github.com/user-attachments/assets/e8b291e2-225b-4a5d-9693-4ba2864f8f00">
